src/routes/chats.js

```diff
@@ -1,25 +1,229 @@
 module.exports = (() => {
 	const express = require('express');
 	const router = express.Router();
-	// const path = require('path');
-	// const chatController = require(path.join(__dirname, '../controllers/chat.controller'));
+	const path = require('path');
+	const chatController = require(path.join(__dirname, '../controllers/chat.controller'));
+	// bring in middleware to verify user's membership in chat
+	const membershipMiddleware = require(path.join(__dirname, '../middlewares/membership'));
 
-	// must have some middleware to verify user is authenticated
-	// AND user making the API call is part of the given chat
+	/**
+	* @api {get} /chat Get chats user is in
+	* @apiName GetChats
+	* @apiGroup Chat
+	*
+	* @apiHeader {String} token Authorization Bearer Token.
+	*
+	* @apiSuccess {Object[]} - List of chats.
+	* @apiSuccess {String} -.id Chats id.
```

Comment:
Same as above I think.
```suggestion
	* @apiSuccess {Number} id Chats ID.
```